csi.proto

```diff
@@ -945,6 +993,17 @@ message ControllerServiceCapability {
       // Indicates the SP supports the
       // ListVolumesResponse.entry.published_nodes field
       LIST_VOLUMES_PUBLISHED_NODES = 10;
+
+      // Indicates the SP supports the
+      // ListVolumesResponse.entry.volume_health field
+      LIST_VOLUMES_VOLUME_HEALTH = 11 [(alpha_enum_value) = true];
+
+      // Indicates the SP supports the GetVolume RPC
+      GET_VOLUME = 12 [(alpha_enum_value) = true];
+
+      // Indicates the SP supports the
+      // GetVolumeResponse.volume_health field
+      VOLUME_HEALTH = 13 [(alpha_enum_value) = true];
```

Comment:
Ditto. Should we change to `VOLUME_CONDITION`?

Reply:
Done